# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../gnubg/ftgl-build.patch
# Copyright (C) 2006 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- gnubg-0.14.3/board3d/font3d.cpp.vanilla	2006-12-28 12:43:40.000000000 +0000
+++ gnubg-0.14.3/board3d/font3d.cpp	2006-12-28 12:44:38.000000000 +0000
@@ -29,9 +29,9 @@
 /* Using ftgl in a static libaray */
 #define FTGL_LIBRARY_STATIC
 
-#include "FTGLPolygonFont.h"
-#include "FTGLOutlineFont.h"
-#include "FTFont.h"
+#include <FTGL/FTGLPolygonFont.h>
+#include <FTGL/FTGLOutlineFont.h>
+#include <FTGL/FTFont.h>
 
 /* Avoid FTGLOutlineFont::Render() as expensive to call repeatedly */
 class MyOutlineFont : public FTGLOutlineFont
